src/components/Typography/BodyStylised.js

```diff
@@ -7,9 +7,10 @@ const BodyStylised = styled.p`
   color: ${props => props.theme.colors.secondaryText};
   font-family: 'PT Mono', sans-serif;
   font-weight: 400;
-  font-size: ${remcalc(15)};
-  line-height: ${remcalc(24)};
-  padding: ${remcalc(13)} 0 ${remcalc(11)} 0;
+  font-size: ${({ small = false }) => (small ? remcalc(13) : remcalc(15))};
```

Comment:
Perhaps we could have a single block which says "if `small=false` then apply all these styles"? Feels a bit more readable to me.

What do you think?

Reply:
updated